### DISCUSSION TITLE
Error 429 again

### DISCUSSION THREAD

**Discussion by sea220-dev** [bimmerconnected/bimmer_connected#674] [Q&A]
Hello everybody,

since yesterday I'm getting error 429 again (#629, #626). I was using version 0.16.3. An upgrade to 0.16.4 does'nt solve the issue. I deleted .bimmer_connected.json, but that does'nt help either. 
Any ideas?

thanks Michael

**Comment by rikroe** [comment]
Please see #671, which breaks this library completely at the moment.
